x-pack/plugins/cases/public/components/user_profiles/unknown_user.tsx

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+import { UserAvatar } from '@kbn/user-profile-components';
+import * as i18n from '../../common/translations';
+
+const CaseUnknownUserAvatarComponent: React.FC = () => {
+  return (
+    <UserAvatar
+      user={{ username: '', display_name: i18n.UNKNOWN }}
+      avatar={{ initials: 'U' }}
```

Comment:
Nothing wrong with this approach but just in case you've missed it, you can render an avatar for an unknown user by omitting `user` and `avatar` props:

```jsx
<UserAvatar />
```

Which will render like this:

<img width="56" alt="Screenshot 2022-08-18 at 10 51 27" src="https://user-images.githubusercontent.com/190132/185366578-12a3f2bc-3286-44da-b035-332ae7a61965.png">

Reply:
Oh nice, I'll switch it to that 👍